app/controllers/api/v1/reviews_controller.rb

```diff
@@ -5,18 +5,35 @@ def create
     if user_signed_in?
       # Design descision to be made: will the react element need the wizard?
       if new_review.save
-        render json: { wizard: wizard , review: review_params }
+        render json: { wizard: wizard, review: review_params }
```

Comment:
RESTful convention wise, we wouldn't necessarily return the wizard in the response body when we _create_ a review.